### ISSUE TITLE
Add HomeKit control for fan position for Falcon S DR-HAF004S

### ISSUE THREAD

**Issue by solmssen** [zyonse/homebridge-dreo#13]
**Is your feature request related to a problem? Please describe:**
Hi - this fan works quite well with the plugin, and I'm very grateful for your work. The iOS Dreo app gives "PTZ"-like capabilities (similar to an IP camera) to put the fan at a particular place in its range of motion by sliding the blue dot around in the "3D" control as shown in the attached screenshot. I have no idea if HomeKit exposes this kind of functionality, but if it can move cameras, maybe it can do the fan?

**Describe the solution you'd like:**
It would be very helpful to be able to tell Siri to "place the fan at 45 Left 5 Up" or something like that without having to open the app.

**Describe alternatives you've considered:**
The Dreo Alexa functionality does allow you to turn on horizontal and vertical oscillation to move the fan and you can watch it and stop it where you want it if you're good at timing your command, but it is not true "put the fan here please" kind of functionality.

![Screenshot 2023-06-25 at 12 35 58 PM](https://github.com/zyonse/homebridge-dreo/assets/3509208/b65e174c-5e61-4c92-8647-726a0b232ab8)

**Comment by zyonse** [comment]
Are there any other fans that have this kind of control with homekit? I'm trying to figure out if this is possible.

**Comment by solmssen** [comment]
Hi - I found this one, but I don't know if it's positionable like the Dreo, or just on and off. It also appears to not be released yet. https://eu.smartmiglobal.com/de/pages/smartmi-circulating-fan. I honestly don't know if this is even worth your time, what I found is that the fan loses track of where it is pointed pretty quickly and has to be recalibrated often. This may be one of those things where it's easier to just get up and move it. I looked and it looks like even PTZ IP cameras have to be controlled through their apps - HomeKit just doesn't expose that functionality I guess. A feature for the next version of the API I hope...
